workflow/util/util.go

```
@@ -353,7 +354,7 @@ func ResumeWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string, nodeFi
 
 			err = packer.DecompressWorkflow(wf)
 			if err != nil {
-				log.Fatal(err)
+				log.Fatalf("unable to decompress workflow: %s", err)
```

uh - panic? surely we should return the error